nf_core/components/install.py

```diff
@@ -264,3 +267,25 @@ def clean_modules_json(self, component, modules_repo, modules_json):
                             self.component_type, component, repo_to_remove, modules_repo.repo_path
                         )
                         return component_values["installed_by"]
+
+    def check_alternate_remotes(self, modules_json):
+        """
+        Check whether there are previously installed components with the same org_path but different remote urls
+        Log warning if multiple remotes exist.
+
+        Return:
+            True: if problematic components are found
+            False: if problematic components are not found
+        """
+        alternate_remotes = False
+        modules_json.load()
+        for repo_url, repo_content in modules_json.modules_json.get("repos", dict()).items():
+            for dir in repo_content.get(self.component_type, dict()).keys():
+                if dir == self.org and repo_url != self.modules_repo.remote_url:
+                    alternate_remotes = True
+        if alternate_remotes:
+            warn_msg = f"Multiple module remotes are used with the same org_path '{self.org}': {', '.join(alternate_remotes)}. This may result in reinstalled modules from the wrong remote."
+            log.warning(warn_msg)
+            return True
+        else:
+            return False
```

Comment:
`alternate_remotes` will be overwritten in this loop :) 
```suggestion
                    return True
        return False
```

Reply:
i am not sure i follow how it would be overwritten, in any case i simplified the logic and hopefully should be ok now!